opswat/opswat/opswat.py

```diff
@@ -57,12 +57,15 @@ async def scan(self, payload: Payload, request: Request) -> WorkerResponse:
             'apikey': self.apikey,
             'content-type': 'application/octet-stream',
             'filename': payload.results.payload_meta.extra_data.get(
-                'filename', get_sha1(payload.content)
-            ),
+                'filename1', get_sha1(payload.content).encode()
```

Comment:
@mlaferrera  What is this change for?  Is it unrelated to the PR?

Reply:
damn, that was from me testing to ensure both `filename` and `get_sha1` worked, but I failed to revert the filename key back. I just pushed an update to fix that. Thanks for bringing it to my attention!